packages/mdc-snackbar/_variables.scss

```diff
@@ -15,9 +15,9 @@
 //
 
 // Hard coded since the color is not present in any palette.
-$mdc-snackbar-background-color: #323232;
-$mdc-snackbar-background-color-on-dark: #fafafa;
-$mdc-snackbar-foreground-color: white;
+$mdc-snackbar-background-color: #323232 !default;
```

Comment:
I would confirm with designer if the [guideline spec](https://material.io/guidelines/components/snackbars-toasts.html#snackbars-toasts-specs) is recommendation on snackbar.